config/manager/manager.yaml

```diff
@@ -36,6 +36,7 @@
       labels:
         control-plane: controller-manager
     spec:
+      automountServiceAccountToken: true
```

Comment:
operator does not work w/o SA

Reply:
> operator does not work w/o SA

Can you add this comment in the `manager.yaml` file, next to the related line? Just for our reference.

Reply:
```suggestion
      # Required because the operator does not work without a Service Account Token
      automountServiceAccountToken: true # NOSONAR
```

Reply:
Testing it locally with sonarLint plugin for Goland I can confirm - nothing like //NOSONAR or #pragma  worked for me on Dockerfiles, yaml or Go files
Also it is not clear where to put such annotation in case if Sonar ask to add something (which is most of our cases), there are no "end of the line of the issue".

I can not test if my changes works on sonarCloud as well b/c it does not show Issues for PR (only for main branch) I'll talk to @kim-tsao about sonarCloud settings and exceptions (as mentioned above) .